config/config.go

```diff
@@ -72,7 +90,8 @@ func load0(configDir string) (*Config, error) {
 	// as we haven't yet unified with the runtime config.
 	ctx := cuecontext.New()
 	insts := load.Instances([]string{"."}, &load.Config{
-		Dir: configDir,
+		Dir:  configDir,
+		Tags: o.tags,
```

Comment:
Good find! I didn't realise this would be so straightforward.

Reply:
Yes – helpful to be able to read the existing implementation in the `cue` source ;-)